Gems/ROS2/Code/Source/Camera/CameraSensorConfiguration.h

```diff
@@ -19,7 +19,8 @@ namespace ROS2
         AZ_TYPE_INFO(CameraSensorConfiguration, "{386A2640-442B-473D-BC2A-665D049D7EF5}");
         static void Reflect(AZ::ReflectContext* context);
 
-        float m_verticalFieldOfViewDeg = 90.0f; //!< Vertical field of view of camera sensor.
+        float m_horizontalFieldOfViewDeg = 86.0f; //!< Horizontal field of view of camera sensor.
+        float m_verticalFieldOfViewDeg = 57.0f; //!< Vertical field of view of camera sensor.
```

Comment:
The defaults should be referring to some representative, real, common device here. If this is the case, ignore the comment (resolve it).